addons/binding/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/handler/HomematicThingHandler.java

```diff
@@ -397,4 +403,15 @@ public void handleConfigurationUpdate(Map<String, Object> configurationParameter
             logger.error("Error setting thing properties: {}", ex.getMessage(), ex);
         }
     }
+
+    private void validateConfiguration() {
+        boolean addressFromAutodiscovery = getThing().getProperties().containsKey(PROPERTY_ADDRESS);
```

Comment:
If everything is done right, the handler should actually have no chance to see whether something came through auto-discovery or was defined in some other way - so this logic here looks suspicious to me.